weave/trace/weave_client.py

```diff
@@ -440,6 +452,28 @@ def __init__(
 
     @trace_sentry.global_trace_sentry.watch()
     def save(self, val: Any, name: str, branch: str = "latest") -> Any:
+        """Do not call directly, use weave.publish() instead.
```

Comment:
can we mark this deprecated? or `def save` -> `def _save`? Docstring is good but still will probably be used directly.

Reply:
It is not "deprecated" per-se. But since it is in the open, we can't really change it at this point. I think the right thing is to fix the issues in the list (: